podman/domain/system.py

```diff
@@ -51,11 +50,23 @@ def login(
             email: Registry account email address
             registry: URL for registry access. For example,
                 https://quay.io/v2
-            reauth: If True, refresh existing authentication. Default: False
```

Comment:
docker-py caches previous authorization results and with reauth False the code just reads the cache rather than round trip to the service.

Rather than delete the keyword, change the text to inform developers it is ignored.